setup.py

```diff
@@ -61,6 +61,7 @@
 ]
 
 install_requires = [
+    'celery>=3.1.16,<4.0.0',
```

Comment:
Well, what I meant is that the tests should pass no matter which celery version we use, docs are a different thing 😞

cc @hachreak